server/polar/customer_portal/endpoints/subscription.py

```diff
@@ -104,6 +105,72 @@ async def get(
     return subscription
 
 
+class SubscriptionChargePreviewResponse(TypedDict):
+    base_amount: int
+    subtotal_amount: int
+    tax_amount: int
+    total_amount: int
+
+
+@router.get(
+    "/{id}/charge-preview",
+    summary="Preview Next Charge For Active Subscription",
+    response_model=SubscriptionChargePreviewResponse,
+    responses={404: SubscriptionNotFound},
```

Comment:
Perhaps add the tag `APITag.private`, so it's not (yet) considered as a public API.